mms/model_service_worker.py

```diff
@@ -15,6 +15,7 @@
 
 # pylint: disable=redefined-builtin
 
+import cProfile
```

Comment:
Why import this module in production? Have a method which can be called when "Benchmark" is set to True and import this from there.

Reply:
Moved import to cProfile initialization